src/Core/src/Core/IMenuElement.cs

```diff
@@ -1,9 +1,12 @@
-﻿using Microsoft.Maui.Graphics;
-
-namespace Microsoft.Maui
+﻿namespace Microsoft.Maui
 {
 	public interface IMenuElement : IElement, IImageSourcePart, IText
 	{
+		/// <summary>
+		/// Represents a shortcut key for a MenuItem.
+		/// </summary>
+		IAccelerator Accelerator { get; }
```

Comment:
I assume we want to support a "no accelerator"?

```suggestion
		IAccelerator? Accelerator => null
```

This suggestion is to use DIM and allow nulls.